lib/TH/generic/THTensorMath.c

```diff
@@ -1995,6 +1995,7 @@ LAB_IMPLEMENT_BASIC_FUNCTION(floor,floor)
 LAB_IMPLEMENT_BASIC_FUNCTION(round,round)
 LAB_IMPLEMENT_BASIC_FUNCTION(abs,fabs)
 LAB_IMPLEMENT_BASIC_FUNCTION(neg,-)
+LAB_IMPLEMENT_BASIC_FUNCTION(inv,TH_inv)
```

Comment:
Just wondering, shouldn't we try to be consistent in here ? [In your cutorch PR](https://github.com/torch/cutorch/pull/306/files#diff-13ee6e0e8f659bdf5f14261f7bee76a5R54), you used `1.0f /`.